code/modules/events/processor_overload.dm

```diff
@@ -29,6 +29,8 @@
 /datum/round_event/processor_overload/start()
 	for(var/obj/machinery/telecomms/processor/P in GLOB.telecomms_list)
 		if(prob(10))
+			if (!atom_of_interest)

```

Comment:
Shouldn't this be set even if prob fails ? I guess less interesting this way.

Reply:
I mean if prob fails theres nothing to orbit